### PR TITLE
Shorten long lines fourth 100

### LINKED ISSUE #12034: [Tracking] Extremely long XML comments that need to be wrapped
There are XML comments that far exceed the recommended 120 characters on one line, they should be wrapped to multiple lines. This should probably be done after #12028 to avoid merge conflicts. There length should be agreed upon before work starts, 120 characters seems too long to me.  This is part of #11984.

### PR DESCRIPTION
**Pull request by paul1956**
Partly Fixes #12034

## Proposed changes
- Shorten another batch of lines

## Customer Impact
- None

## Regression? 

- No

## Risk
- No code changes

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12078)

### CODE REVIEW

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CollectionCodeDomSerializer.cs [hunk 1]
```diff
@@ -132,13 +138,17 @@ protected bool MethodSupportsSerialization(MethodInfo method)
         //      If the collection has a method called AddRange, we will call that, providing an array.
         //      If the collection has an Add method, we will call it repeatedly.
         //      If the collection is an IList, we will cast to IList and add to it.
-        //      If the collection has no add method, but is marked with PersistContents, we will enumerate the collection and serialize each element.
-        // Check to see if there is a CodePropertyReferenceExpression on the stack. If there is, we can use it as a guide for serialization.
+        //      If the collection has no add method, but is marked with PersistContents,
+        //      we will enumerate the collection and serialize each element.
+        // Check to see if there is a CodePropertyReferenceExpression on the stack.
+        // If there is, we can use it as a guide for serialization.
```
Comment (Epica3055) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CollectionCodeDomSerializer.cs:144]:
```suggestion
        // If the collection has no add method, but is marked with PersistContents,
        // we will enumerate the collection and serialize each element.
        // Check to see if there is a CodePropertyReferenceExpression on the stack.
        // If there is, we can use it as a guide for serialization.
```

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CollectionCodeDomSerializer.cs [hunk 2]
```diff
@@ -132,13 +138,17 @@ protected bool MethodSupportsSerialization(MethodInfo method)
         //      If the collection has a method called AddRange, we will call that, providing an array.
         //      If the collection has an Add method, we will call it repeatedly.
         //      If the collection is an IList, we will cast to IList and add to it.
```
Comment (Epica3055) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CollectionCodeDomSerializer.cs:140]:
```suggestion
        // If the collection has a method called AddRange, we will call that, providing an array.
        // If the collection has an Add method, we will call it repeatedly.
        // If the collection is an IList, we will cast to IList and add to it.
```

**Comment by paul1956** [comment]
> Just curicurious Why capitalize the microsoft in https://learn.microsoft.com/. I tried to go https://learn.Microsoft.com/ then it became https://learn.microsoft.com/
@Epica3055 

```
    ///  <see href="https://learn.microsoft.com/windows/win32/api/commdlg/ns-commdlg-printdlgw#members">
    ///    Read more on learn.Microsoft.com.
```

There are 2 parts, https://learn.microsoft.com does not need to be capitalized. But learn.microsoft.com  on the second line does or you get spelling error.
image

See https://github.com/dotnet/winforms/issues/11984#issuecomment-2336452490

**Comment by Tanya-Solyanik** [comment]
@paul1956  - please resolve the conflicts.

**Comment by paul1956** [comment]
> Just curicurious Why capitalize the microsoft in https://learn.microsoft.com/. I tried to go https://learn.Microsoft.com/ then it became https://learn.microsoft.com/

PR #12105 address all the spelling issues of microsoft by changing to http. I will undo here

**Comment by paul1956** [comment]
> @paul1956 - please resolve the conflicts.

@Tanya-Solyanik Done

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomComponentSerializationService.CodeDomSerializationStore.ComponentListCodeDomSerializer.cs [hunk 3]
```diff
@@ -350,7 +356,7 @@ private bool ResolveName(IDesignerSerializationManager manager, string name, boo
                             }
                         }
 
-                        // Sometimes components won't be in either the statements table or the expressions table, for example, this occurs for resources  during undo/redo. In these cases the component should be resolvable by the manager. Never do this when we have been asked by the serialization manager to resolve the name;  otherwise we may infinitely recurse.
+                        // Sometimes components won't be in either the statements table or the expressions table, for example, this occurs for resources during undo/redo. In these cases the component should be resolvable by the manager. Never do this when we have been asked by the serialization manager to resolve the name;  otherwise we may infinitely recurse.
```
Comment (LeafShi1) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomComponentSerializationService.CodeDomSerializationStore.ComponentListCodeDomSerializer.cs:359]: This comment needs to be displayed on multiple lines

Comment (paul1956) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomComponentSerializationService.CodeDomSerializationStore.ComponentListCodeDomSerializer.cs:359]: @LeafShi1 not all long lines are fixed in every PR. This issue is fixed in PR #12106

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ToolStripTemplateNode.cs [hunk 4]
```diff
@@ -1733,7 +1762,11 @@ protected override void WndProc(ref Message m)
                     if (_owner._addItemButton is null)
                     {
                         // only adding patterns to _miniToolStrip associated with MenuStrip or ContextMenu
-                        // m.Result = AutomationInteropProvider.ReturnRawElementProvider(Handle, m.WParam, m.LParam, (IRawElementProviderSimple)(new TransparentToolStripUiaProvider(this)));
+                        // m.Result = AutomationInteropProvider.ReturnRawElementProvider(
+                        //     Handle,
+                        //     m.WParam,
+                        //     m.LParam,
+                        //     (IRawElementProviderSimple)(new TransparentToolStripUiaProvider(this)));
```
Comment (LeafShi1) [inline src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ToolStripTemplateNode.cs:1769]: The code snippet has been commented out, please consider delete it directly.

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/UndoEngine.UndoUnit.cs [hunk 5]
```diff
@@ -396,9 +403,9 @@ protected virtual void UndoCore()
...
                     // Property change is an undo event that may have an unknown side effect if changing the property results in other property changes (for example, reparenting a control removes the control from its former parent) .A property change undo event must have all BeforeUndo methods called before any Undo method is called. To do this, we have a property on UndoEvent called CausesSideEffects.
-                    // As we run through UndoEvents, consecutive events that return true from this property are grouped so that their BeforeUndo methods are all called before their Undo methods. For events that do not have  side effects, their BeforeUndo and Undo are invoked immediately.
+                    // As we run through UndoEvents, consecutive events that return true from this property are grouped so that their BeforeUndo methods are all called before their Undo methods. For events that do not have side effects, their BeforeUndo and Undo are invoked immediately.
```
[the remark this answers is not in the document]
Comment (LeafShi1) [reply src/System.Windows.Forms.Design/src/System/ComponentModel/Design/UndoEngine.UndoUnit.cs:408]: Looks like this issue also fixed in PR https://github.com/dotnet/winforms/pull/12106.

File: src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ToolStripTemplateNode.cs (hunk 4 above)
Comment (paul1956) [reply src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/ToolStripTemplateNode.cs]: Done

**Review by LeafShi1 — APPROVED** [review]

**Review by lonitra — APPROVED** [review]
Thank you

**Pull request merged**